Directory.Build.props

```diff
@@ -6,7 +6,9 @@
     <AssemblyProduct>Python.NET</AssemblyProduct>
     <LangVersion>10.0</LangVersion>
     <IsPackable>false</IsPackable>
-    <Version>$([System.IO.File]::ReadAllText("version.txt"))</Version>
+    <FullVersion>$([System.IO.File]::ReadAllText("version.txt"))</FullVersion>
+    <VersionPrefix>$(FullVersion.Split('-', 2)[0])</VersionPrefix>
```

Comment:
Can you fix the monthly NuGet script right away? Should be easy to try locally

Reply:
Currently the version looks like this: `3.0.0-preview2022-04-11` 3.0.0 being the prefix and the rest is the suffix.

Reply:
I'm not sure I understand, with this change I get exactly the result you would expect. When specifying the suffix on the command-line, it overrides whatever suffix (if one was set) from the `version.txt` file.